doc/adoc/user_guide.adoc

```diff
@@ -90,6 +90,19 @@ If your system is not configured for it, it is recommended that at least
 one of the users on the system has a `~/.ssh/authorized_keys` file with a
 private key accessible by the person executing the system upgrade.
 
+== Upgrade Pre-Checks
+The suse-migration-pre-checks package contains the `suse-migration-pre-checks`
+script that checks for possible incompatibilities when doing a migration.
+
+These incompatibilities include:
+
+ Encrypted file systems
+ Invalid repository types
+
+This script is run during the install of SLES15-Migration. It can
+also be run anytime using `/usr/bin/suse-migration-pre-checks`
+
+This script
```

Comment:
maybe a blank line under this
as it is now, it produces
![image](https://user-images.githubusercontent.com/3913611/162723189-8e77c619-8c98-47b0-81f6-15a308987e90.png)

Reply:
thanks I will fix and render the doc to make usre it looks ok.

Reply:
How does this look @jesusbv ?

![image](https://user-images.githubusercontent.com/9559808/162761020-52a8ea10-d6a8-40ff-bb85-469fb89f2e96.png)